### PR TITLE
Fix: Button border radius

### LINKED ISSUE #310: Chrome applying styles to buttons on Mac
This is a new issue that suddenly appeared. In Chrome, a border radius is being applied to button elements as shown in the screenshot below. 'Documentation' is not using the button tag so it's rendering correctly. If you go into dev tools and change any <a> links to button, a border radius is applied. 

screen shot 2017-11-21 at 11 36 49 am

This is only a chrome issue. Here is Firefox:

screen shot 2017-11-21 at 11 39 55 am

Safari

screen shot 2017-11-21 at 11 39 50 am

This is only happening in Chrome on my Mac and the style is loading via the user agent stylesheet. I'm running the same version of Chrome on my Surface and the style is not being loaded at all there. 

Just need to apply a border-radius: 0 to buttons.

### PR DESCRIPTION
**Pull request by ststimac**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/contributing/pull-requests/#commitmessages)

## Short description of the change(s)

Chrome's user agent stylesheet adds a border radius to button elements. Overriding those styles to align with original design.

Fix #310

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "stylelint": "^8.2.0"
   },
   "hexo": {
-    "version": "3.4.0"
+    "version": "3.3.8"
```
Comment (ststimac) [inline package.json]: Is this supposed to be 3.3.8 or 3.4.0? It looks like it was changed back to 3.3.8 in the master branch?

Comment (alrra) [reply package.json]: Running `npm run build` locally changes it to `3.4.0`. I will amend your commit to include the change.

Comment (ststimac) [reply package.json]: Thank you ^_^

**Comment by ststimac** [comment]
@alrra I forgot to put a space before a comment in my CSS file so the build failed. Do you want me to make that update before you amend the commit?

File: package.json (hunk 1 above)
Comment (alrra) [reply package.json]: I'll also [fix the linting problem](https://travis-ci.org/sonarwhal/sonarwhal.com/builds/306069030#L2275).

**Comment by alrra** [comment]
>  I forgot to put a space before a comment in my CSS file so the build failed. Do you want me to make that update before you amend the commit?

Don't worry, I'll take care of it.

File: package.json (hunk 1 above)
Comment (ststimac) [reply package.json]: Thank you, again.

**Pull request closed without merging**